sjb/config/test_cases/azure_build_node_image_rhel_310.yml

```diff
@@ -31,6 +34,10 @@ actions:
   - type: "host_script"
     title: "move secrets to remote host"
     script: |-
+      set +x
+      export KUBECONFIG=/var/lib/jenkins/.azure/kubeconfig
+      echo $(oc get secret aws-reg-token --template={{.data.token}} |base64 -d) > /var/lib/jenkins/.azure/registry
```

Comment:
I assume we will use this eventually for running PR tests. If `oc get secret` fails then the token will get overwritten and jobs running in parallel beyond this step will not have a token to use?

Reply:
Whats true. How unreliable is our CI cluster for this? Do we need to build some contingency in this now? I would like to assume perfect world for now and if we see a lot of flakes we can add some checks.

Reply:
The CI cluster is mostly reliable but the minimal fix for avoiding this flake comes at no cost.
```bash
NEW_TOKEN=$(oc get secret aws-reg-token --template={{.data.token}} | base64 -d)
if [[ -n $NEW_TOKEN ]]; then
  echo $NEW_TOKEN > /var/lib/jenkins/.azure/registry
fi
```